tools/sqlite3_api_wrapper/sqlite3_api_wrapper.cpp

```diff
@@ -593,6 +621,24 @@ int sqlite3_bind_text(sqlite3_stmt *stmt, int idx, const char *val, int length,
 	}
 }
 
+int sqlite3_bind_blob(sqlite3_stmt *stmt, int idx, const void *val, int length, void (*free_func)(void *)) {
+	if (!val) {
+		return SQLITE_MISUSE;
+	}
+	if (free_func && ((ptrdiff_t)free_func) != -1) {
+		free_func((void *)val);
```

Comment:
Why call the free_func immediately after binding? Wouldn't this give a use-after-free exception? I suggest creating the `Value::BLOB` first before calling free.

Reply:
Oh yes, I copy code from sqlite3_bind_text but didn't get it right. Thanks!